### ISSUE TITLE
Question about float support - float4

### ISSUE THREAD

**Issue by jindrapetrik** [airsdk/Adobe-Runtime-Support#3406]
Hello,
I'd like to ask a question. I don't know whether it is a right place.

In AIR 51 you added float support.

My question is - why did you not include a float4 support?

In AVMPLUS source code here:
https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/AbcParser.cpp#L1332
the ABC format with float support contained both float and float4 entries (there was no special flag for float4 reading)

In AIR 51 it seems that only floats are written, float4 entries are ignored.

It is a little confusing for the SWF readers, that previously, float4 was there and now isn't.

Just for the clarification: I am the author of JPEXS Free Flash decompiler, that's why I ask.

It's not a big deal - I will drop the float4 support from the decompiler.

I am just curious.

It would make a more sense if you wrote float4Count = 0 everytime, rather than not include the section at all.

Thank you in advance for any response.

**Comment by jindrapetrik** [comment]
Thank you for your answer!
Yes, I was not sure the float4 was ever used - I did not find any compiler for it. Everything I know about it is from the public AVMPlus source. I only added it's support incase someone has this way compiled ABC.
Nevermind, you're right, if it wasn't ever released before, then it can be changed to anything.
I understand that changing it again to float4Count = 0 would break the compatibility for AIR 51 again.

I updated the decompiler to ignore the float4 now, it should help for issues like #3349.
